Makefile

```diff
@@ -1,20 +1,35 @@
 PREFIX ?= /usr/local
-MAN_PREFIX ?= ${PREFIX}/man
+BINDIR ?= $(PREFIX)/bin
+MANDIR ?= $(PREFIX)/man
```

Comment:
`man hier` on my system indicates man pages should be installed either to `/usr/local/man` or `/usr/share/man`, with `/usr/man` being deprecated in favor of `/usr/share/man`.
Looking around for prior art on this, I notice that autotools uses (haven't read the rest of the script, so I'm not sure why they're single-quoting here)
```bash
bindir='${exec_prefix}/bin'
sbindir='${exec_prefix}/sbin'
libexecdir='${exec_prefix}/libexec'
datarootdir='${prefix}/share'
datadir='${datarootdir}'
sysconfdir='${prefix}/etc'
sharedstatedir='${prefix}/com'
localstatedir='${prefix}/var'
runstatedir='${localstatedir}/run'
includedir='${prefix}/include'
oldincludedir='/usr/include'
docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
infodir='${datarootdir}/info'
htmldir='${docdir}'
dvidir='${docdir}'
pdfdir='${docdir}'
psdir='${docdir}'
libdir='${exec_prefix}/lib'
localedir='${datarootdir}/locale'
mandir='${datarootdir}/man'
```

Reply:
> man hier on my system indicates man pages should be installed either to /usr/local/man or /usr/share/man, with /usr/man being deprecated in favor of /usr/share/man.

The only way I can see to handle this is some sort of conditional in the Makefile, unless I'm missing out on something.

Reply:
One hack I've seen here to avoid conditionals comes from `batsignal`:
```make
MANPREFIX.$(PREFIX)=$(PREFIX)/share/man
MANPREFIX./usr/local=/usr/local/man
MANPREFIX.=/usr/share/man
MANPREFIX=$(MANPREFIX.$(PREFIX))
```

Reply:
That is very clever! I guess in our context this would become:

```
MANDIR.$(PREFIX) = $(PREFIX)/share/man
MANDIR./usr/local = /usr/local/man
MANDIR. = /usr/share/man
MANDIR ?= $(MANDIR.$(PREFIX))
```

?